packages/web-component-tester/bin/wct

```diff
@@ -14,9 +14,7 @@ process.title = 'wct';
 
 resolve('web-component-tester', {basedir: process.cwd()}, function(error, path) {
   var wct = path ? require(path) : require('..');
-  var promise = wct.cli.run(process.env, process.argv.slice(2), process.stdout, function (error) {
-    process.exit(error ? 1 : 0);
-  });
+  var promise = wct.cli.run(process.env, process.argv.slice(2), process.stdout);
```

Comment:
on the next line... do you know why this call might not return a promise?

Reply:
Not a clue.